.travis.yml

```diff
@@ -18,6 +21,5 @@ before_script:
 - sh -e /etc/init.d/xvfb start
 - jdk_switcher use oraclejdk8
 - yarn start > /dev/null 2>&1 &
-- while ! curl --silent http://localhost:4444/wd/hub > /dev/null 2>&1; do sleep 1; done
```

Comment:
Still wait for the server though....maybe not the selenium server, but the dev server at least.